src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -403,6 +455,26 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
         {profilerOverhead ? (
           <MetaOverheadStatistics profilerOverhead={profilerOverhead} />
         ) : null}
+        {this.props.hasProfileExtraInfo ? (
+          <div className="metaInfoRow">
+            <button
+              type="button"
+              className="moreInfoButton photon-button photon-button-default photon-button-micro"
+              onClick={this._handleMoreInfoButtonClick}
```

Comment:
For accessibility, it would be good to use the attribute `aria-expanded` with the value `true` or `false` reflecting the state.

Note: MDN says to use `aria-controls` too. My understanding was that it's optional especially when the expanded element is right after it. But a quick check on various resources told me that it's recommended as well. See these resources:
* https://www.w3.org/WAI/ARIA/apg/example-index/disclosure/disclosure-faq
* https://adrianroselli.com/2020/05/disclosure-widgets.html

So let's include it as well because it's fairly easy to use.

Reply:
But I should be able to omit them if I use the `<details>` element?

Reply:
Yeah I think so